expression/builtin_time_test.go

```diff
@@ -1427,6 +1427,48 @@ func (s *testEvaluatorSuite) TestSecToTime(c *C) {
 	}
 }
 
+func (s *testEvaluatorSuite) TestConvertTz(c *C) {
+	tests := []struct {
```

Comment:
case `{"2004-01-01 12:00:00", "GMT", "+00:00", true, ""}` covers `! fmatch && tmatch`, and I add another case to cover `fmatch && !tmatch?`